### ISSUE TITLE
FacebookLogin response is lost when device is locked/unlocked when permission dialog is showing

### ISSUE THREAD

**Issue by ssh-pm** [facebook/facebook-ios-sdk#1347]
## Checklist

- [ *] I've updated to the latest released version of the SDK
- [ *] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-ios-sdk/issues)
- [ *] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [* ] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [ *] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Xcode Version: `11.3.1 (11C504)`
- Swift Version: `X.X.X` (if issue is Swift related)
- Installation Platform & Verison: `[Cocoapods|Carthage|Manual]` version `1.8.4`

## Goals

I want to see the response from Facebook SDK

## Expected Results

I expected to get some response from FacebookSDK, possibly User Cancel response when device is unlocked

## Actual Results

App doesn't receive any response

## Steps to Reproduce

- Integrate Facebook SDK ios with the app
- Provided custom login button (not using bulit-in Facebook SDK Login button)
- Click on Login button
- The Permission dialog shows up
- Lock the device
- Unlock the device
- See that permission dialog is disappeared
- Facebook doesn't send any response to the app

## Code Samples & Details

Please provide a code sample, as well as any additional details, to help us track down the issue. If you can provide a link to a test project that allows us to reproduce the issue, this helps us immensely in both the speed and quality of the fix.

**Note:** Remember to format your code for readability:

```Objective C
    [FBSDKProfile enableUpdatesOnAccessTokenChange:TRUE];
    FBSDKLoginManager *facebookLoginManager = [FBSDKLoginManager new];
    if([FBSDKAccessToken isCurrentAccessTokenActive]){
        [facebookLoginManager logOut];
    }else {
        [facebookLoginManager logInWithPermissions:@[@"email", @"public_profile"]
                                fromViewController:self handler:^(FBSDKLoginManagerLoginResult * _Nullable result, NSError * _Nullable error) {
                                    if(error != nil){
                                        NSLog(@"Login with error: %@",error.localizedDescription);
                                        return;
                                    }
                                    if(result.isCancelled){
                                        NSLog(@"Login is cancelled");
                                    }else{
                                        NSLog(@"Login with token: %@",result.token);
                                    }
                                }];
    }
```

**Comment by robtimp** [comment]
Fixed by commit a3957f8fd0c36f07281048d7fba255ad354e6368.

**Comment by ssh-pm** [comment]
which version of Facebook ios sdk this fix will be available?
